src/os/webgl/launchlayercompareperfdialog.js

```diff
@@ -0,0 +1,37 @@
+goog.module('os.webgl.launchLayerComparePerformanceDialog');
+
+const Promise = goog.require('goog.Promise');
+const {launchConfirm} = goog.require('os.ui.window.ConfirmUI');
+
+
+/**
+ * Launch a dialog warning users of the risks in using 2D with lots of data.
+ *
+ * @return {!Promise}
+ */
+const launch2DPerformanceDialog = function() {
```

Comment:
Minor nit. I would suggest putting this in `layercompareui.js` because it's not particularly useful outside of the `launchLayerCompare` function.